src/main/java/org/teamresistance/frc/Robot.java

```diff
@@ -25,6 +25,8 @@
   private final FlightStick rightJoystick = Hardware.HumanInterfaceDevices.logitechAttack3D(1);
   private final FlightStick coJoystick = Hardware.HumanInterfaceDevices.logitechAttack3D(2);
 
+  private OpticalFlow opFlow = new OpticalFlow();
```

Comment:
This line won't be necessary if you intend to use the optical flow from the `IO` class, which is the approach I suggest.